runtime/queries/scala/highlights.scm

```diff
@@ -0,0 +1,202 @@
+; CREDITS @stumash (stuart.mashaal@gmail.com)
+
+;; variables
+
+(identifier) @variable
```

Comment:
Move this to the end of the file. tree-sitter takes the first matching query, and this matches and overrides a lot of the later queries.

It significantly improved highlighting for me: 
![pos](https://user-images.githubusercontent.com/1372918/146548243-534058c1-65be-4fe1-95c5-3b1a93a4d8c7.png)